index.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(input) {
 
     var text = $e.text();
     var klass = $e.attr('class') || '';
-    var lang = klass.split('lang-').filter(id);
+    var lang = klass.split('language-').filter(id);
```

Comment:
I think this should support both options so we don't break backwards compatibility.

Reply:
good call, update submitted